frontend/src/metabase/public/containers/PublicOrEmbeddedQuestion/PublicOrEmbeddedQuestion/PublicOrEmbeddedQuestion.tsx

```diff
@@ -48,6 +51,14 @@ export const PublicOrEmbeddedQuestion = ({
   const [parameterValues, setParameterValues] = useState<ParameterValuesMap>(
     {},
   );
+
+  const { setFont } = useEmbedFont();
+  useEffect(() => {
+    const { font } = parseHashOptions(location.hash) as DashboardUrlHashOptions;
```

Comment:
I think you could use `EmbeddingDisplayOptions`. As this type has 3 specific dashboard options that aren't present on questions
https://github.com/metabase/metabase/blob/0d33cce6228ba25d11232e36cc4932f7c91e0a66/frontend/src/metabase/dashboard/types/hash-options.ts#L9-L13